### PR TITLE
2017 april newsletter

### PR DESCRIPTION
**Pull request by imrehg**
Work in progress.

### CODE REVIEW

File: src/data/posts.json [hunk 1]
```diff
@@ -1,32 +1,47 @@
...
+    "title": "The Raspberry Pi and other SBCs are the IBM PCs of our time",
+    "text": "From educational project to industry disruptor.",
+    "link": "https://resin.io/blog/the-raspberry-pi-and-other-sbcs-are-the-ibm-pcs-of-our-time-from-educational-project-to-industry-disruptor/",
+    "image": "http://cdn1.alphr.com/sites/alphr/files/6/14/raspberry_pi_b_2_0_0.jpg",
+    "button": "Read the post"
   },
   {
-    "title": "Work with us!",
+    "title": "Come Work with Us!",
     "tags": ["jobs"],
```
Comment (craig-mulligan) [inline src/data/posts.json]: If I remember correctly tags are used to make links to the tags on our blog eg. resin.io/tag/jobs? If that's still the case this should be removed.

Comment (imrehg) [inline src/data/posts.json]: Indeed, I was wondering about that! :)

File: src/data/featured.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-	"title": "ElectronJS with Raspberry Pi Made Easy As Pie",
-	"tags": ["tutorial"],
-	"text": "Showcasing the opportunities of ElectronJS with a Raspberry Pi to create digital signage and interactive displays, including example projects!",
-	"link": "https://resin.io/blog/electronjs-the-ultimate-guide/",
-	"image": "http://i.imgur.com/8GWASHi.jpg",
-	"button": "Deploy it"
+	"title": "ResinOS 2.0 Released in Production",
```
Comment (alisondavis17) [inline src/data/featured.json]: Let's edit all the titles to start with a verb that applies to the audience - e.g. "Deploy resinOS 2.0 to your fleet"

File: src/data/posts.json [hunk 3]
```diff
@@ -1,32 +1,47 @@
 [
   {
-    "title": "New board support: the UP board",
-    "tags": ["announcement"],
-    "text": "The UP board is x86-based with a Raspberry Pi compatible 40-pin connector, designed for 'professional makers' who need a reliable, high-performing device to build and grow their projects.",
-    "link": "https://resin.io/blog/up-board-support/",
-    "image": "https://resin.io/blog/content/images/2017/03/UPSlide3Right-EVT-3.png",
+    "title": "LEGO Macintosh Classic with e‑paper display",
```
Comment (alisondavis17) [inline src/data/posts.json]: Similarly, let's change the title to "Build a LEGO Macintosh Classic with e-paper display"

File: src/data/posts.json [hunk 4]
```diff
@@ -1,32 +1,47 @@
...
+    "tags": ["project"],
+    "text": "An awesome, detailed community project of a wi-Fi enabled LEGO Macintosh Classic running Docker on a Raspberry Pi Zero with an e‑paper display",
+    "link": "https://jann.is/lego-macintosh-classic/",
+    "image": "https://c2.staticflickr.com/4/3762/33583084842_b2089a98f1_z.jpg",
+    "button": "Check it out"
+  },
+  {
+    "title": "Cellular connectivity with resinOS 2.0, 3G dongles, and SORACOM SIM cards",
```
Comment (alisondavis17) [inline src/data/posts.json]: Change title to "Add cellular connectivity to your fleet with resinOS 2.0, 3G dongles, and SORACOM SIM cards"

File: src/data/posts.json [hunk 5]
```diff
@@ -1,32 +1,47 @@
...
+    "tags": ["tutorial"],
+    "text": "With this tutorial complete, you'll be able to ship out devices that work anywhere in the world.",
```
Comment (alisondavis17) [inline src/data/posts.json]: This description doesn't preview what's in the post, let's make it more specific to the post

File: src/data/posts.json [hunk 6]
```diff
@@ -1,32 +1,47 @@
...
     "button": "Read more"
   },
   {
-    "title": "Case Study: Open ROV",
-    "tags": ["case study"],
-    "text": "Using resin.io on underwater remote controlled drones.",
-    "link": "https://vimeo.com/208753228",
-    "image": "https://i.vimeocdn.com/video/624266745_1280x720.jpg",
-    "button": "Dive in"
+    "title": "Resin SDK v6 released",
```
Comment (alisondavis17) [inline src/data/posts.json]: Change title to "Use the resin SDK to manage your fleet - v6 just released!"

File: src/data/posts.json [hunk 7]
```diff
@@ -1,32 +1,47 @@
...
+    "tags": ["announcement"],
+    "text": "OS metadata, browser support, custom locations, and API changes.",
+    "link": "https://resin.io/blog/resin-sdk-v6-released/",
+    "image": "http://www.caves-corner.co.uk/ekmps/shops/cavescorner1/images/dominoes-3760-[4]-127-p.jpg",
+    "button": "See changes"
+  },
+  {
+    "title": "Advanced Device Provisioning Workflow for Large Fleets: Preloading and Pre-provisioning",
```
Comment (alisondavis17) [inline src/data/posts.json]: Change title to "Preload applications and pre-provision devices in your fleet"

File: src/data/events.json [hunk 8]
```diff
@@ -1,23 +1,41 @@
...
+		"link": "http://makerfaireathens.com/en/"
+	},
+	{
+		"title": "Resin.io Meetup Amsterdam",
+		"date": "2017 May 11",
+		"location": "Amsterdam, The Netherlands",
+		"link": "https://www.meetup.com/resin-io/events/239037689/"
+	},
+	{
+		"title": "Resin.io Workshop & Hackathon",
```
Comment (alisondavis17) [inline src/data/events.json]: Can we call this out higher up in the newsletter, as a separate item?

**Comment by alisondavis17** [comment]
what's the cover photo this month @imrehg?

**Comment by alisondavis17** [comment]
let's also mention blog4swag?

File: src/data/posts.json (hunk 5 above)
Comment (imrehg) [inline src/data/posts.json]: Hm, that line was taken verbatim from the post. Will rewrite then.

File: src/data/posts.json (hunk 4 above)
Comment (imrehg) [inline src/data/posts.json]: Changed.

File: src/data/posts.json (hunk 3 above)
Comment (imrehg) [inline src/data/posts.json]: Changed.

File: src/data/featured.json (hunk 2 above)
Comment (imrehg) [inline src/data/featured.json]: Changed.

**Comment by imrehg** [comment]
@alisondavis17 the cover photo is the `image` tag in the `featured.json` file, ie. for resinOS 2.0 release. Since the blogpost had no image, using this:



Will add Blog4Swag, reorg the items.

File: src/data/events.json (hunk 8 above)
Comment (imrehg) [inline src/data/events.json]: Yes, added as 2nd item.

File: src/data/posts.json (hunk 7 above)
Comment (imrehg) [inline src/data/posts.json]: Done

File: src/data/posts.json (hunk 6 above)
Comment (imrehg) [inline src/data/posts.json]: Done

**Comment by imrehg** [comment]
I had to cut some posts, as the email was too long and being clipped in Gmail - just clipping above "events", so by default none of that and below show, just have to click a button to show the whole email. That's not great, so I cut 2 posts that feel less important from the newsletter point of view, and now it still clips, but just after "Events" (i.e only hides the footer by default). This seems to be a reasonable compromise.

@alisondavis17 @craig-mulligan

**Pull request merged**